tests/system/stats/prometheus/prometheus_stats_test.py

```diff
@@ -74,11 +74,11 @@ def test_prometheus_stats(self):
             import urllib2
             contents = urllib2.urlopen("http://localhost:9303/metrics").read()
 
-        self.assertIn(b'# TYPE opencensus_request_count_view counter',
+        self.assertIn(b'# TYPE opencensus_request_count_view_total counter',
```

Comment:
The latest Prometheus client automatically append "_total" suffix to count metrics.